docs/learn-sapio/src/ch01-01-installation.md

```diff
@@ -6,24 +6,24 @@ Sapio should work on all platforms, but is recommend for use with Linux (Ubuntu
 Follow this quickstart guide to get going.
 
 1.  Get [rust](https://rustup.rs/) if you don't have it already.
-1.  Add the wasm target by running the below command in your terminal:
+1.  Add the wasm target and nightly toolchain by running the below command in your terminal:
 ```bash
-rustup target add wasm32-unknown-unknown
```

Comment:
i'm going to revert this change in follow up, since folks may not want a full rustup update